motor_blog/text/slugify.py

```diff
@@ -1,6 +1,9 @@
+import sys
 import re
 from unicodedata import normalize
 
+reload(sys)
+sys.setdefaultencoding('utf-8');
```

Comment:
Because utf-8 is compatible with ascii. I use chinese to edit my blog so it's better to change default coding to utf 8.for convenient, I think it's better to change it. 
Maybe there is another way can generate chinese slug, I wil try  another way to do it.Could you give me some suggestions ?thanks